packages/client/package.json

```diff
@@ -30,6 +30,7 @@
     "@spectrum-css/tag": "^3.1.4",
     "@spectrum-css/typography": "^3.0.2",
     "@spectrum-css/vars": "^3.0.1",
+    "@spectrum-web-components/progress-bar": "^0.7.0",
```

Comment:
It would be better to use `"@spectrum-css/progressbar": "^3.0.1",` instead of the `spectrum-web-components` - this is more consistent with what we currently have

Reply:
I removed this as I didnt use Adobe , i used the internal BBUI implementation